packages/Modal/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@welcome-ui/modal",
+  "version": "1.0.0",
+  "description": "Modal from Reakit with a really nice theme 👀",
+  "main": "dist/modal.cjs.js",
+  "module": "dist/modal.es.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/WTTJ/welcome-ui.git"
+  },
+  "keywords": [
+    "modal",
+    "dialog",
+    "design-system",
+    "react",
+    "reakit",
+    "styled-components",
+    "styled-system",
+    "ui-library",
+    "ui",
+    "ux",
+    "jungle",
+    "welcome",
+    "WTTJ"
+  ],
+  "author": "WTTJ <tech@wttj.co>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/WTTJ/welcome-ui/issues"
+  },
+  "homepage": "https://github.com/WTTJ/welcome-ui#readme",
+  "dependencies": {
+    "@welcome-ui/box": "^1.5.2",
+    "@welcome-ui/button": "^1.5.2",
+    "@welcome-ui/icons.cross": "^1.5.2",
+    "@welcome-ui/shape": "^1.5.2",
+    "@welcome-ui/utils": "^1.4.0",
+    "reakit": "^1.0.0-beta.7"
+  },
+  "peerDependencies": {
+    "@xstyled/styled-components": "^1.15.0",
+    "@xstyled/system": "^1.14.0",
+    "react": "^16.10.2",
+    "react-dom": "^16.10.2",
+    "styled-components": "^4.0.0",
+    "prop-types": "^15.7.2"
```

Comment:
dependencies should be in alphabetical order. :D

Reply:
🙈